subsys/bluetooth/host/hci_core.h

```diff
@@ -189,6 +189,11 @@ extern struct bt_dev bt_dev;
 extern const struct bt_conn_auth_cb *bt_auth;
 #endif /* CONFIG_BT_SMP || CONFIG_BT_BREDR */
 
+struct net_buf *hci_cmd_create(u16_t opcode, u8_t param_len);
+int hci_cmd_send(u16_t opcode, struct net_buf *buf);
+int hci_cmd_send_sync(u16_t opcode, struct net_buf *buf,
+				 struct net_buf **rsp);
```

Comment:
Minor: please align right after the `(` of the first line (i.e. tabs + 0-7 spaces)